src/Shared/Themes/BlazorBoilerplate.CommonUI/Shared/Layouts/MainLayout.razor

```diff
@@ -6,8 +6,8 @@
     <MatDrawer @bind-Opened="@_navMenuOpened">
         <header class="drawer-header">
             <div class="drawer-logo">
-                <img alt="Blazor Boilerplate" class="logo-img" src="_content/BlazorBoilerplate.CommonUI/images/blazorboilerplate-dark.svg" title="Blazor Boilerplate">
-                <a class="miniHover" href="/">Blazor Boilerplate</a>
+                <img alt="BlazorBoilerplate" class="logo-img" src="_content/BlazorBoilerplate.CommonUI/images/logo-dark.svg" title="BlazorBoilerplate">
```

Comment:
Is there a reason for removing the space "Blazor Boilerplate" - > "BlazorBoilerplate"  I would prefer to keep the space there.

Reply:
To allow string replace in templating. It is anyway not so important, it is to have 'donet new blazorboileplate -n MyApp' create a solution with MyApp instead of Blazor Boilerplate as a title.

Reply:
fix it in https://github.com/enkodellc/blazorboilerplate/pull/262/commits/cffc516f47d9e36a5e36137f171325772a3e6349